templates/uber-development.ini.erb

```diff
@@ -74,6 +74,14 @@ shirt_sales_enabled = <%= @shirt_sales_enabled %>
 show_affiliates = <%= @show_affiliates %>
 student_discount = <%= @student_discount %>
 
```

Comment:
> It doesn't have a default if an event isn't running the hotel plugin.

Oh right, I forgot that this was already split out into the ``hotel`` plugin, but that we haven't yet puppetized our plugin config.  Gotcha, that all makes sense then.